slack_bolt/listener/thread_runner.py

```diff
@@ -160,7 +160,9 @@ def run_ack_function_asynchronously():
                     self._start_lazy_function(lazy_func, request)
 
             # await for the completion of ack() in the async listener execution
-            while ack.response is None and time.time() - starting_time <= 3:
+            print(str(starting_time))
+            while ack.response is None and time.time() - starting_time <= listener.acknowledgement_timeout:
+                print("we are sleeping")
```

Comment:
```suggestion
            while ack.response is None and time.time() - starting_time <= listener.acknowledgement_timeout:
```

🪵 question: Are these outputs for testing or should we change to `logger` printing?

Reply:
Ohh good catch yes this was for testing 🚀